src/internal/orders/sales-order-time-range.ts

```diff
@@ -1,26 +1,19 @@
 import Joi = require("joi");
 import {
-  SalesOrderPaging as SalesOrderPagingPOJO,
   SalesOrderStatus,
-  SalesOrderTimeRange as ISalesOrderTimeRange,
-  TimeRangePOJO
+  SalesOrderTimeRange as ISalesOrderTimeRange
 } from "../../public";
 import { hideAndFreeze, TimeRange, TimeRangeBase, _internal } from "../common";
 import { SalesOrderPaging } from "./sales-order-paging";
-
-export interface SalesOrderTimeRangePOJO extends TimeRangePOJO {
-  paging?: SalesOrderPagingPOJO;
-  statusMappings?: {
-    [key: string]: SalesOrderStatus;
-  };
-}
```

Comment:
This interface is still needed.  Importing the public interface and aliasing it is not the same.